charts/hedera-mirror-grpc/values.yaml

```
@@ -30,6 +30,28 @@ image:
 
 imagePullSecrets: []
 
+ingress:
+  annotations:
+    traefik.ingress.kubernetes.io/router.middlewares: '{{ include "hedera-mirror-grpc.namespace" . }}-{{ include "hedera-mirror-grpc.fullname" . }}@kubernetescrd'
+  enabled: true
+  hosts:
+    - host: ""
+      paths:
+        - "/com.hedera.mirror.api.proto.ConsensusService"
```

Would be best to avoid the burden of configuring every path if we have no reason to block any.
In case of grpc, i believe we can open up all paths(?).
Is is possible to specify something like `*` here?

---

No, these paths are global as all paths are served on the same port. So you need some way to route rest vs grpc, hence above.

---

Well, to clarify I can put `/` here instead and Traefik will route to longer, more specific paths like `/grafana` and `/api/v1` first. If the path doesn't match those it will redirect all remaining traffic to GRPC service where it may or may not be found. But think I prefer failing fast at proxy layer instead with more specific paths then potentially overloading service layer.